app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -270,7 +270,7 @@ def hash_for_pdf
       end
 
       # line 20a
-      if @xml_document.at("PensAnnuitAndIraWithdraw") && Flipper.enabled?(:show_retirement_ui)
+      if @xml_document.at("PensAnnuitAndIraWithdraw")
```

Comment:
@mluedke2 neat thing - adding the explicit Flipper disabled tests allowed me to clean up this line of code, because the test actually confirms that if the flipper is disabled, the XML will be nil, which means we don't need an extra feature flag check here in the PDF. pretty cool!

Reply:
oh nice! 👍